### PR TITLE
Use correct style for re-exported imports

### PR DESCRIPTION
**Pull request by AMDmi3**
## What do these changes do?

Use correct style for re-exported imports, makes `mypy --strict` happy.

## Are there changes in behavior for the user?

No

## Related issue number

#3868

## Checklist

- [x] I think the code is well written
- [ ] Unit tests for the changes exist (no new code and functional changes though, so should be covered by existing tests)
- [ ] Documentation reflects the changes
- [x] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
- [x] Add a new news fragment into the `CHANGES` folder

### CODE REVIEW

**Comment by AMDmi3** [comment]
Please give me some more time, I was not aware `from foo import (bar as baz)` syntax is possible. Will redo the patches that way, that will also fix flake8.

**Comment by AMDmi3** [comment]
Done. I'm not sure how to fix isort though - for me it (the same version, 4.3.21) produces a huge diff all over the module.

File: aiohttp/__init__.py [hunk 1]
```diff
@@ -176,6 +186,7 @@
     'ThreadedResolver',
     # signals
     'Signal',
+    # streams
```
Comment (webknjaz) [inline aiohttp/__init__.py]: It's unrelated

**Comment by webknjaz** [comment]
You should probably do whatever isort tells you to.

File: aiohttp/__init__.py (hunk 1 above)
Comment (AMDmi3) [reply aiohttp/__init__.py]: It's related to the imports cleanup, and makes __all__ consistent with imports here.

Comment (webknjaz) [reply aiohttp/__init__.py]: It's unrelated to re-exports.

File: CHANGES/3868.bugfix [hunk 2]
```diff
@@ -0,0 +1 @@
+Use correct style for re-exported imports, makes mypy --strict mode happy.
```
Comment (webknjaz) [inline CHANGES/3868.bugfix]:
```suggestion
Use correct style for re-exported imports, makes mypy ``--strict`` mode happy.
```

File: aiohttp/__init__.py (hunk 1 above)
Comment (AMDmi3) [reply aiohttp/__init__.py]: ok

**Comment by asvetlov** [comment]
I very sorry, your PR has conflicts after merging #3632

Could you please fix these conflicts?

**Comment by AMDmi3** [comment]
Updated

**Comment by asvetlov** [comment]
Thanks!

**Pull request merged**